features/0434-outofband/README.md

```diff
@@ -0,0 +1,448 @@
+
+#### Message Type: `https://didcomm.org/outofband/%VER/invitation`
+
+```jsonc
+{
+  "@type": "https://didcomm.org/outofband/%VER/invitation",
+  "@id": "<id used for context as pthid>",
+  "label": "Faber College",
+  "purpose-cd": "issue-vc",
+  "purpose": "To issue a Faber College Graduate credential",
+  "protocols": [
+      "https://didcomm.org/didexchange/1.0",
+      "https://didcomm.org/connections/1.0"
+      ]
+  "service": ["did:sov:LjgpST2rjsoxYegQDRm7EL"]
+}
+```
+
+The items in the message are:
+
+- `@type` - the DIDComm message type
+- `@id` - the unique ID of the message. The ID should be used as the **parent** thread ID (`pthid`) for the response message, rather than the more common thread ID (`thid`) of the response message. This enables multiple uses of a single out of band message.
+- `label` - [optional] a self-attested string that the receiver may want to display to the user, likely about who sent the out of band message.
+- `purpose-cd` - [optional] a self-attested code the receiver may want to display to the user or use in automatically deciding what to do with the out of band message.
+- `purpose` - [optional] a self-attested string that the receiver may want to display to the user about the context-specific purpose of the out of band message.
+- `protocols` - array of protocols in the order of preference of the sender that the receiver can use in responding to the message. As implied by the message type of invitation, these are not arbitrary protocols but rather protocols that result in the establishment of a connection.
+- `service` - an item that is the equivalent of the service block of a DIDDoc that the receiver is to use in responding to the message. Additional details below.
+
+Both the `purpose-cd` and `purpose` fields should be used with the [localization service decorator](../0043-l10n/README.md). The two fields are to enable both human and machine handling of the out of band message. `purpose-cd` is to specify a generic, protocol level reason for the connection (e.g. issue verifiable credential, request proof, etc.) that is suitable for machine handling and possibly human display, while `purpose` provides context specific guidance, targeting mainly a person controlling the receiver's agent.
+
+While the _receiver_ is expected to respond with an initiating message from the chosen protocol using a specified service, the receiver may be able to respond by reusing an existing connection. Specifically, if a connection they have was created from an out of band `invitation` from the same public DID of a new `invitation`, the receiver **SHOULD** use the existing connection in responding to the `invitation`. The selected `protocol` being used by the receiver may have a specific message type to use for this purpose.
+
+> **To Do**: Define the list of `purpose-cd`
```

Comment:
Suggest a registry instead of a "list"

Reply:
Once a registry mechanism has been defined, we can move the list to that.  For now, I put a table in the message catalog with a few initial values. I don't think they are perfect, but that will get the discussion started.  I hope that discussion will be a new issue and not part of this PR :-).